themes/default/content/docs/guides/adopting/from_kubernetes.md

```diff
@@ -392,9 +391,11 @@ let k8s = require("@pulumi/kubernetes");
 
 // Deploy the latest version of the stable/wordpress chart.
```

Comment:
Several comments still refer to `stable/wordpress`

Reply:
Updated.